rhel/Dockerfile

```diff
@@ -45,19 +45,22 @@ RUN curl -sSL https://packages.instana.io/Instana.gpg -o /tmp/Instana.gpg && \
     curl -L https://github.com/hairyhenderson/gomplate/releases/download/v3.6.0/gomplate_linux-amd64-slim -o /usr/bin/gomplate && \
     chmod 755 /usr/bin/gomplate && \
     rm -rf /tmp/* /etc/yum.repos.d/Instana-Agent.repo /etc/yum.repos.d/docker.repo && \
-    microdnf clean all
+    microdnf clean all && \
+    curl https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 -o /usr/bin/jq && \
```

Comment:
When downloading, we should at least verify the checksum (should do that with the `gomplate` as well...)

`echo "af986793a515d500ab2d35f8d2aecd656e764504b789b66d7e1a0b727a124c44 /usr/bin/jq" | sha256sum -c - `

Reply:
And I would move these lines to a bit above, next to `gomplate` install, so it's clear those installs belong together.